lib/rules/base.js

```diff
@@ -75,6 +76,54 @@ module.exports = class BaseRule {
     return name === this.ruleName || DEPRECATED_RULES.get(name) === this.ruleName;
   }
 
+  _determineConfigForSeverity(config) {
```

Comment:
Let's make this and the method just below utility methods (not on the base class)